src/index.html

```diff
@@ -5,8 +5,415 @@
+      >
+    <link rel="preconnect" href="https://fonts.gstatic.com" ><link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#home"
+      >
+        <img
+          class="header__logo-moyo"
+          src="./images/logo.png"
+          alt="logo_moyo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            > Apple
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a
+              href="#samsung"
+              class="nav__link"
+            > Samsung
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            > Smartphones
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a
+              href="#laptops & computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            > Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            > Gadgets
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a
+              href="#tablets"
+              class="nav__link"
+            > Tablets
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a
+              href="#photo"
+              class="nav__link"
+            > Photo
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a
+              href="#video"
+              class="nav__link"
+            > Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="catalog__content">
+        <div
+        class="card catalog__card"
+        data-qa="card"
+        >
+          <div class="imagin-container">
+            <img class="imagin-container__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+             >
+          </div>
+
+          <p class="card__name-product">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card-rating">
+            <div class="stars-block">
+              <div class="stars-block__star stars-block__star--active"></div>
+              <div class="stars-block__star stars-block__star--active"></div>
+              <div class="stars-block__star stars-block__star--active"></div>
+              <div class="stars-block__star stars-block__star--active"></div>
+              <div class="stars-block__star stars-block__star--active"></div>
```

Comment:
Cool, but what if i need set 5 stars for this item, and for the next, we need to set 3 stars?